### DISCUSSION TITLE
Dynamic routed page critical issue.

### DISCUSSION THREAD

**Discussion by ryanBuzzni** [vercel/next.js#41437] [Help]
From about 10:00 today, a dynamically routed page chunk file issue has occurred on the server and the service is not available.
.e.g.
https://saja.market/_next/static/chunks/pages/product/%5Bpid%5D-7e2f8caef162da38.js
net::ERR_ABORTED 404

When I changed it to https://saja.market/_next/static/chunks/pages/product/[pid]-7e2f8caef162da38.js, it opened

Please, help me

**Comment by MoonsuCha** [comment]
<img width="550" alt="image-29" src="https://user-images.githubusercontent.com/8610430/195966171-aad78f1b-c20b-4f90-bf7f-b75afaa21c93.png">
We are experiencing the same problem. What is the situation??

**Comment by pr-o** [comment]
Hi, @ryanBuzzni

Thanks to **_@syi0808_**'s insight, we have resolved this issue by adding a webpack plugin. 
The plugin renames the `dynamic route page file` to an appropriate filename. (e.g. `%5Babc%5D.js` => `[abc].js`)

Refer to the following gist: https://gist.github.com/pr-o/6a279a8c263f02c4f1623526d23dfa3b

**Comment by ijjk** [comment]
Hi, this should be resolved now, please refer to our status page for tracking https://www.vercel-status.com/incidents/tnrdjcqfbxd7

**Reply by pr-o** [reply]
Thanks for the update 👍
